integration/run/testdata/volume/Acornfile

```diff
@@ -3,5 +3,12 @@ containers: {
 		build: "."
 		dirs: "/test": "./files"
 		dirs: "/external": "external"
+		dirs: "/tmp": "volume://my-data"
 	}
 }
+
+volumes: {
+	"my-data": {
+		size: "1G"
+	}
+}
```

Comment:
What does this default to when not defined? Worth writing a test for when this is not defined as well?

Reply:
There are tests for this case. It defaults to 10G.